tests/helpers/http.go

```diff
@@ -194,20 +200,20 @@ func AssertReportResponsesEqual(t testing.TB, expected, got []byte) {
 
 	err := JSONUnmarshalStrict(expected, &expectedResponse)
 	if err != nil {
-		log.Error().Msg("Error unmarshalling expected value")
+		log.Error().Msg(errorUnmarshallingExpectedValue)
```

Comment:
It's a nitpick: probably with the `err` value logged by `FailOnError` would be enough,

Reply:
I'll better leave it as it is, because it's not very clear which value is failed.

Reply:
+1 for FailOnError. You can modify it like this:

```golang
// FailOnError wraps result of function with one argument
func FailOnError(t testing.TB, err error, msgAndArgs ...interface{}) {
	// assert.NoError is used to show human readable output
	assert.NoError(t, err, msgAndArgs...)
	// assert.NoError doesn't stop next test execution which can cause strange panic because
	// there was error and some object was not constructed
	if err != nil {
		t.Fatal(err)
	}
}
```